from get import check

try:
    check()
except Exception as e:
    print(e)